Source/ACE/Entity/DebugObject.cs

```diff
@@ -60,32 +60,11 @@ public DebugObject(ObjectGuid guid,  BaseAceObject baseAceObject)
             this.GameData.Value = baseAceObject.Value;
             this.GameData.ItemCapacity = baseAceObject.ItemsCapacity;
 
-            // Put is in for Ripley - these are the fields I want to write that he was concerned with.
-            if ((this.Type & (ObjectType.Creature | ObjectType.LifeStone | ObjectType.Portal)) == 0)
-            {
-                // because this comes from PCAP data - on create we are not animating.
-                this.PhysicsData.AnimationFrame = 0x65;
```

Comment:
that is base animation frame.   With my if code above, I wanted to exclude lifestones, portals and creatures.   It should be cleaned up in the database.   If the pcap saw this equipped and id it the animation frame is set for wielded.   If spawned or dropped items are not 0x65 (101) they are at about a 70% angle from the surface.   at 0x65 they lay flat on the ground.   That if statement for this debug code.